docs/app/data/components-page.json

```diff
@@ -1508,25 +1508,36 @@
             "link": "tables",
             "sections": [
                 {
-                    "title": "Column Resizing",
```

Comment:
Looks like Column Resizing was accidentally removed. There are two copies of Column Sorting in the documentation.

Reply:
Thanks, that should now be added back in